sdk/storage/src/common/crypt.cpp

```diff
@@ -275,4 +389,823 @@ namespace Azure { namespace Storage {
 
 #endif
 
+  static constexpr uint64_t poly = 0x9A6C9329AC4BC9B5ULL;
```

Comment:
Good thing is that track 1 proved the below code's integrity. Otherwise, I don't know how to maintain it.

Reply:
abstruse mathematical principles behind it. Don't try to comprehend it. just use it.